src/hubAuth.ts

```diff
@@ -101,7 +101,9 @@ export const testkitHubAuth = (homeDir: string, authStrategy: DevhubAuthStrategy
     }
     return;
   }
-  if (authStrategy === 'AUTH_URL') {
+  // the transferExistingAuthToEnv method will transfer the TESTKIT_HUB_USERNAME env var to be TESTKIT_AUTH_URL
+  // so we can auth with the URL for both
+  if (authStrategy === 'AUTH_URL' || authStrategy === 'REUSE') {
```

Comment:
`REUSE` is supposed have your testkit sessions use existing auth that you specify, and "figure it out" based on your envs

So it *could* get the AUTH_URL.  But you could have also given it the username of a hub that you have already auth'd via JWT.  In which case, you'd want it to auth with that, too.